tests/empty.py

```diff
@@ -0,0 +1,10 @@
+import torch
+
+#simple torch.empty
+a = torch.empty((2, 3))
+print(a)
+
+# output to another tensor
+b = torch.ones(1)
+torch.empty((3, 3), out=b)
```

Comment:
This will generate arbitrary set of float values. You can set `dtype = torch.int64`. That should give you values of zeros I believe

Reply:
https://pytorch.org/docs/stable/generated/torch.empty.html. You check this for syntax